tests/integrations/import/sst_service.rs

```diff
@@ -108,6 +108,13 @@ fn test_ingest_sst() {
     let resp = import.ingest(&ingest).unwrap();
     assert!(!resp.has_error());
 
+    // Set crc32 == 0 and length != 0
+    meta.set_crc32(0);
+    meta.set_length(data.len() as u64);
+    ingest.set_sst(meta.clone());
+    let resp = import.ingest(&ingest).unwrap();
+    assert!(!resp.has_error());
```

Comment:
```
[2020-03-25T10:02:13.351Z] test import::sst_service::test_ingest_sst ... thread 'apply-1' panicked at '[region 1] 1 ingest uuid: 50F9339230554764AD40201F4348BE93 range { start: 00 end: 64 } length: 1634 cf_name: "default" region_id: 1 region_epoch { conf_ver: 1 version: 1 }: EngineTraits(Other("[components/engine_rocks/src/import.rs:22]: Storage Engine read metadata from /tmp/test_cluster4qkiE7/db/import-sst/50f93392-3055-4764-ad40-201f4348be93_1_1_1.sst: Os { code: 2, kind: NotFound, message: \"No such file or directory\" }"))', components/raftstore/src/store/fsm/apply.rs:1410:17
```

The SST file is automatically deleted after ingestion ended.

Reply:
oh I see